modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js

```diff
@@ -204,6 +207,9 @@ const TranslationAdminContent = ({
 													)
 												}
 												symbol="trash"
+												title={Liferay.Language.get(
+													'delete'
```

Comment:
Why "deleted" instead of "remove"?

Reply:
Same as the other, it seems that it is used in other places